internal/builders/go/pkg/provenance.go

```diff
@@ -33,7 +33,7 @@ const (
 	buildType              = "https://github.com/slsa-framework/slsa-github-generator-go@v1"
```

Comment:
shall we update this string? The builder's fullpath contains both the URI and the version/tag.

Reply:
It's ok for this PR.